python/doc/examples/meta_modeling/kriging_beam_arbitrary_trend.ipynb

```diff
@@ -124,6 +124,22 @@
     "Y_train = model(X_train)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We notice here that input sample contains huge values. Thus we start by transforming data :"
```

Comment:
Good thinking! Examples need to show how to transform the data if the KrigingAlgorithm no longer does it.

Reply:
Indeed. Probably more examples are required. I am trying to find the best way for

Reply:
I do not get the point of getting the transformation without using it in the example. Should'nt the example show how to actually use the transformation?

Reply:
we apply it to the data (cell 15)

Reply:
we apply it to the data (cell 15)